### PR TITLE
update current year(copyright)

### PR DESCRIPTION
**Pull request by Arhell**
Resolves https://github.com/nunit/docs/issues/620

### CODE REVIEW

File: docs/custom_template/partials/footer.tmpl.partial [hunk 1]
```diff
@@ -8,7 +8,12 @@
         <a href="#top">Back to top</a>
       </span>
       {{{_appFooter}}}
-      {{^_appFooter}}<span>Generated by <strong>DocFX</strong></span> | Copyright (c) 2018 The NUnit Project - Licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank">CC BY-NC-SA 4.0</a>{{/_appFooter}}
+      {{^_appFooter}}<span>Generated by <strong>DocFX</strong></span> | Copyright (c) 2018-<span id="currentYear"></span> The NUnit Project - Licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank">CC BY-NC-SA 4.0</a>{{/_appFooter}}
     </div>
   </div>
 </footer>
+
```
Comment (SeanKilleen) [inline docs/custom_template/partials/footer.tmpl.partial:15]: I think there might be a separate place for the footer script includes. Let me confirm that

Comment (Arhell) [reply docs/custom_template/partials/footer.tmpl.partial:15]: Updated.
Here is an example by which I corrected
https://github.com/dotnet/docfx/blob/dev/src/docfx.website.themes/default/styles/main.js

made a build, everything works.

**Comment by SeanKilleen** [comment]
@Arhell Thank you for the contribution! Please update your PR description: 

> image

From `#620` to `Resolves #620` so that this PR will automatically close the issue when merged.

To update, you can use the menu to the side:

> image

I'll check out the code now.

**Comment by SeanKilleen** [comment]
ℹ️ As a general side note / tip in case it's helpful: you may want to avoid force-pushing branches, as it can get you into some trouble. 

Many small commits is fine; in the end, we'll squash and merge it into one commit, so there's no need to force push.

**Comment by SeanKilleen** [comment]
Ah, I apologize -- I just noticed this is a draft PR and not yet ready for review. I'll pause until you're ready. 👍

**Comment by Arhell** [comment]
> Ah, I apologize -- I just noticed this is a draft PR and not yet ready for review. I'll pause until you're ready. 👍

Done

**Comment by Arhell** [comment]
I usually add an issue link here. (If there is access and the ability to do it). Then, after the merge, the issue will automatically close too.

**Comment by SeanKilleen** [comment]
> I usually add an issue link here. (If there is access and the ability to do it). Then, after the merge, the issue will automatically close too.

Ah, I see! Thanks for pointing that out to me. I'm not used to that being in the UI, and my muscle memory expects the text. It's fine as is. 👍

**Review by SeanKilleen — APPROVED** [review]
👍 Thanks for another contribution!

**Pull request merged**